packages/apollo-server-lambda/src/ApolloServer.ts

```diff
@@ -123,7 +143,9 @@ export class ApolloServer extends ApolloServerBase {
         return callback(null, {
           body: '',
           statusCode: 204,
-          headers: corsHeaders,
+          headers: {
+            ...requestCorsHeaders,
```

Comment:
@abernix, `Headers` cannot be spread like this, it would appear. An alternative would be:

```
headers: {
    ...Object.fromEntries(requestCorsHeaders.entries())
}
```

Same for the cases below. Looks like this could use some test coverage.

Reply:
Thanks very much @dsanders11!

Fixed via https://github.com/apollographql/apollo-server/commit/7e78a2353b9e855d85a7998b06e8f9d4cb1db54d.